docs/site/migration/express-middleware.md

```diff
@@ -6,8 +6,82 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-express-middleware.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3947](https://github.com/strongloop/loopback-next/issues/3947)
-" %}
+Migrating app-level Express middleware from LoopBack 3 (LB3) to LoopBack 4 (LB4)
+requires a wrapper Express app, which will the mount the LB4 and also act as the
+point for mounting the shared Express middleware.
+
+First, make sure you have done the necessary steps to mount your LB3 app on a
+LB4 app as described in
+[this tutorial](https://github.com/strongloop/loopback-next/tree/master/examples/lb3-application#tutorial).
+
+Then, create a directory which will be the wrapper Express app. Move the LB4 app
```

Comment:
Do we want to mention the directory `migrate-express-middelware/` that we use as an example to make it more clear?